components/layout_2020/flexbox/layout.rs

```diff
@@ -848,6 +848,7 @@ impl FlexLine<'_> {
             .map(|(((item, item_result), content_rect), margin)| {
                 let content_rect = flex_context.rect_to_flow_relative(line_size, content_rect);
                 let margin = flex_context.sides_to_flow_relative(*margin);
+                let margin = flex_context.sides_to_flow_relative(*margin).into();
```

Comment:
Remove the previous line, this doesn't compile.

Also, the `margin.into()` below should now be just `margin`.